content/docs/getting-started/installing.md

```diff
@@ -5,64 +5,85 @@ weight: 2
 
-- Chrome Web Store (for Google Chrome, Opera, Brave, Vivaldi, Microsoft Edge (>79), and other Chromium-based browsers)  
+- Chrome Web Store (for Google Chrome, Opera, Brave, Vivaldi, Microsoft Edge Chromium (>79), and other Chromium-based browsers)  
   https://chrome.google.com/webstore/detail/fbeffbjdlemaoicjdapfpikkikjoneco  
 - Add-ons for Firefox (for Mozilla Firefox)  
   https://addons.mozilla.org/firefox/addon/scratch-messaging-extension/  
 
-## From GitHub
-
-[The releases tab](https://github.com/ScratchAddons/ScratchAddons/releases) contains all builds of Scratch Addons, as well as the mirror of the store builds hosted on both servers.
 
 ## From source
 
 ### Cloning the repository
 
-This is the recommended way to install for development purposes. This assumes you have Git installed.
+This is the recommended way to install Scratch Addons for development purposes. This assumes you have Git installed.
 
-To download the repository, simply clone `https://github.com/ScratchAddons/ScratchAddons.git`. Here's an example.
+To download the repository, simply clone `https://github.com/ScratchAddons/ScratchAddons.git`:
 
 ```sh
 $ git clone https://github.com/ScratchAddons/ScratchAddons.git
 ```
+To update Scratch Addons if you cloned it using Git, first `cd` into its folder, and then run the following commands:
+
+```sh
+$ git fetch
+```
+
+```sh
+$ git pull
+```
+
+This will update Scratch Addons and get it ready for code editing. Note that you will need to see the finish updating section [here](https://scratchaddons.com/docs/getting-started/installing/#install-on-google-chrome) if you are using Google Chrome.
+
 
 ### Downloading the zipball
 
-If you don't have Git installed, you can try this method instead.
+If you don't have Git installed, you can try this method instead. Note that you will need to manually repeat this process every time you want to update Scratch Addons.
 
 1. Go to the [repository](https://github.com/ScratchAddons/ScratchAddons) and find the download code button.
 
-![Screenshot 2020-09-04 at 6 41 12 PM](https://user-images.githubusercontent.com/61319150/92291572-f17ef080-eede-11ea-85e8-fda961a56074.png)
+   ![Download code button screenshot](https://user-images.githubusercontent.com/61319150/92291572-f17ef080-eede-11ea-85e8-fda961a56074.png)
 
 2. Click it and select "Download ZIP."
 
-![Screenshot 2020-09-04 at 6 43 57 PM](https://user-images.githubusercontent.com/61319150/92291570-f0e65a00-eede-11ea-80ee-f32e340327d9.png)
+   ![Download ZIP button screenshot](https://user-images.githubusercontent.com/61319150/92291570-f0e65a00-eede-11ea-80ee-f32e340327d9.png)
 
 3. Extract the archive into a folder.
 
 ### Installing on Google Chrome
 
-1. Open `chrome://extensions` to open the Extension Management page by typing it into your address bar.
+1. Open `chrome://extensions` to open the extension management page by typing it into your address bar.
```

Comment:
The Extension Management page should be capitalized as it is the capitalization in Chrome itself.

Reply:
Where in Chrome? The page is called "Extensions" and the menu item that opens it is called "Manage extensions".

Reply:
There is one mention on the Chrome Docs here.

https://developer.chrome.com/docs/extensions/mv3/getstarted/

Reply:
And two mentions in lowercase.

Reply:
@Hans5958 Should I change this back?

Reply:
Yeah, at least that's how I do it